### ISSUE TITLE
flatcam.x86_64-linux is broken

### ISSUE THREAD

**Issue by marcus7070** [NixOS/nixpkgs#116124]
**Describe the bug**
Flatcam fails to build

**To Reproduce**
With flakes: nix build -L github:NixOS/nixpkgs/b3ea24a4144884ab07ed4b8c48fddf4a292f1417#flatcam
Or build from master.
Or just look at Hydra: https://hydra.nixos.org/job/nixpkgs/trunk/flatcam.x86_64-linux

**Additional context**
Complains about libspatialindex_c.

Git bisect is going poorly, seems like there was a version bump in mesa (very long rebuilds) and a period where `python3Packages.pillow` and `python3Packages.shapely` were broken causing an unrelated breakages in Flatcam.

**Notify maintainers**
@trepetti 

Maintainer information:
```yaml
attribute:
flatcam
# a list of nixos modules affected by the problem
module:
```

**Comment by marcus7070** [comment]
Sorry, I wasn't paying attention when I was trying to bisect this. The error is in `python38Packages.Rtree`, Flatcam just depends on it.

**Comment by marcus7070** [comment]
Fixed by #116128.
